SU2_CFD/include/solver_structure.hpp

```diff
@@ -164,7 +164,7 @@ class CSolver {
   CSysVector<su2double> OutputVariables;    /*!< \brief vector to store the extra variables to be written. */
   string* OutputHeadingNames; /*< \brief vector of strings to store the headings for the exra variables */
   
-  CVariable** node;  /*!< \brief Vector which the define the variables for each problem. */
+  CVariable* node;  /*!< \brief Vector which the define the variables for each problem. */
```

Comment:
First step done, I updated the interface of CVariable so that "iPoint" is passed as the first argument to its methods, which means we drop one "*" here. node is still polymorphic but is no longer an array.